lib/fs.js

```diff
@@ -40,6 +41,8 @@ const isWindows = process.platform === 'win32';
 
 const errnoException = util._errnoException;
 
+const NOOP = function() {};
```

Comment:
`noop` in lower case would be better that keeps consistent with https://github.com/nodejs/node/search?utf8=%E2%9C%93&q=NOOP.